docs/developers-guide/substreams-sink-kv.md

```diff
@@ -0,0 +1,231 @@
+---
+description: StreamingFast Substreams sink kv
+---
+
+# `substreams-sink-kv` introduction
+
+## Purpose
+
+This documentation will assist you in using `substreams-sink-kv` to write data from your existing substreams into a key-value store and serve it back through Connect-Web/GRPC.
+
+## Overview
+
+* `substreams-sink-kv` works by reading the output of specially-designed substreams module (usually called `kv_out`) that produces data in a protobuf-encoded structure called `sf.substreams.sink.kv.v1.KVOperations`.
```

Comment:
I would remove the bullet list here personally, might have been copied over from the other tutorial, but it feels weird that an overview is a list.